### PR TITLE
use proper DBI parameter passing to improve security

### PR DESCRIPTION
**Pull request by connortechnology**
remark out call to zmupdate.pl -f

### CODE REVIEW

File: scripts/zmpkg.pl.in [hunk 1]
```diff
@@ -138,7 +138,7 @@ if ( $command =~ /^(?:stop|restart)$/ )
 	}
 }
 
-runCommand( "zmupdate.pl -f" );
+#runCommand( "zmupdate.pl -f" );
```
Comment (kylejohnson) [inline scripts/zmpkg.pl.in:141]: Why is this commented out?

Comment (knight-of-ni) [inline scripts/zmpkg.pl.in:141]: zmupdate -f simply reads info from the dB and turns around and writes it right back to the dB.  It looks like user Stan added this back in 2011, but I don't know what his intent was. This same command was also part of the stock zm init script, but after discussing it with mastertheknife I commented it out (the one in the init file).  

TLDR: This might be the cause of the Conig table corruption issue.

Comment (connortechnology) [inline scripts/zmpkg.pl.in:141]: The thing is, if you are going to modify the Config Table, you have to exclusively lock it so that nothing reads in the meantime.
